PWA/src/plugins/i18n.js

```diff
@@ -552,7 +562,8 @@ const messages = {
   'zh-tw': {
     translate_error: '翻譯錯誤',
     yes: '是',
-    no: '否',
+    no: '關',
+    close: '否',
```

Comment:
Incorrect Chinese translations. Not sure why close was added, but the two translations for no and close should be swapped, no says to close something, and close says no.